circuit_knitting/cutting/qpd/__init__.py

```diff
@@ -18,7 +18,6 @@
     decompose_qpd_instructions,
     WeightType,
     qpdbasis_from_gate,
-    supported_gates,
```

Comment:
Is there a 2-qubit gate in Qiskit that would break our cutting workflow if we tried to cut it?

I'm wondering if we can replace
`if g in supported_gates():`

with
`if g.num_qubits == 2:`

Reply:
I now see your docstring update below. They have to implement `to_matrix`